be/src/runtime/lib_cache.cpp

```diff
@@ -126,10 +126,11 @@ LibCache::LibCacheEntry::~LibCacheEntry() {
 Status LibCache::get_so_function_ptr(
         const std::string& hdfs_lib_file, const std::string& symbol,
         void** fn_ptr, LibCacheEntry** ent, bool quiet) {
+    const std::string& real_symbol = get_real_symbol(symbol);
```

Comment:
you can change argument symbol to origin_symbol, then you needn't change following symbol to real_symbol

Reply:
ok, it is more simple。i will change it.